spec/models/transaction_spec.rb

```diff
@@ -18,48 +18,26 @@
   end
 
   context "value must be between 1 and 99,999,999,999.00 (100 billion minus one)" do
-    it "allows the maximum possible value" do
-      transaction = build(:transaction, fund: fund, value: 99_999_999_999.00)
-      expect(transaction.valid?).to be true
-    end
-
-    it "does not allow a value of less than 1" do
-      transaction = build(:transaction, fund: fund, value: -1)
-      expect(transaction.valid?).to be false
-    end
-
-    it "does not allow a value of more than 99,999,999,999.00" do
-      transaction = build(:transaction, fund: fund, value: 100_000_000_000.00)
-      expect(transaction.valid?).to be false
-    end
+    let(:fund) { build(:fund) }
 
-    it "allows a value between 1 and 99,999,999,999.00" do
-      transaction = build(:transaction, fund: fund, value: 500_000.00)
-      expect(transaction.valid?).to be true
+    it "does allows the maximum possible value" do
+      transaction = build(:transaction, hierarchy: fund, value: 99_999_999_999.00)
+      expect(transaction.valid?).to be_truthy
```

Comment:
How about `expect(transaction).to be_valid` ..?

Reply:
That looks like a mistake that's been re-introduced in a merge 🙊

Reply:
I lost a few tests in a cherry-pick, re-added them now